dracut/30ignition/ignition-ask-var-mount.service

```diff
@@ -7,7 +7,6 @@
 [Unit]
 Description=Ask OSTree to mount /var in initramfs
 DefaultDependencies=false
-After=initrd-switch-root.target
```

Comment:
I don't see any immediate problems with this, but I'd be wary of leaving this unit completely without a lower bound. I'd suggest to make this a `After=basic.target`.